openedx/features/badging/helpers.py

```diff
@@ -116,3 +120,31 @@ def filter_earned_badge_by_joined_team(user, course, earned_badges):
         earned_badge for earned_badge in earned_badges if
         earned_badge.community_id == course_team[TEAM_ROOM_ID_KEY]
     ]
+
+
+def send_user_badge_notification(user, my_badge_url, badge_name):
+    """
+    Send user new badge notification
+    :param user: Notification sender user
```

Comment:
Let's change this to `:param user: User receiving the Notification `

Reply:
user is both sender and receiver in this case.For notifications we need user id to send to and user name (initials will appear in image on left side of notification) from whom the notification is generated.

Reply:
@monofal 
In this case param user is not sender of notification, he will receive the badge.

Reply:
updated